libcontainer/init_linux.go

```diff
@@ -70,6 +70,7 @@ type initConfig struct {
 	RootlessEUID     bool                  `json:"rootless_euid,omitempty"`
 	RootlessCgroups  bool                  `json:"rootless_cgroups,omitempty"`
 	SpecState        *specs.State          `json:"spec_state,omitempty"`
+	Cgroup2Path      string                `json:"cgroup2path,omitempty"`
```

Comment:
json name should be `cgroup2_path` ?

Also, this field needs a comment line to explain that this is the cg2 path of the runtime itself, not the cg2 path for the container being launched.

Reply:
> json name should be cgroup2_path ?

fixed

> to explain that this is the cg2 path of the runtime itself, not the cg2 path for the container being launched

Can you explain in more details? To me, this is exactly the cgroup2 path for container being launched, taken from the container's cgroup manager which is already initialized with the proper config etc.

Reply:
Ah, sorry I misread the code..